clients/src/main/java/org/apache/kafka/common/requests/CreateTokenResponse.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.common.requests;
+
+import org.apache.kafka.common.protocol.ApiKeys;
+import org.apache.kafka.common.protocol.Errors;
+import org.apache.kafka.common.protocol.types.Field;
+import org.apache.kafka.common.protocol.types.Schema;
+import org.apache.kafka.common.protocol.types.Struct;
+import org.apache.kafka.common.security.auth.KafkaPrincipal;
+import org.apache.kafka.common.utils.SecurityUtils;
+
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+import static org.apache.kafka.common.protocol.CommonFields.ERROR_CODE;
+import static org.apache.kafka.common.protocol.CommonFields.THROTTLE_TIME_MS;
+import static org.apache.kafka.common.protocol.types.Type.BYTES;
+import static org.apache.kafka.common.protocol.types.Type.INT64;
+import static org.apache.kafka.common.protocol.types.Type.STRING;
+
+public class CreateTokenResponse extends AbstractResponse {
+
+    private static final String OWNER_KEY_NAME = "owner";
+    private static final String ISSUE_TIMESTAMP_KEY_NAME = "issue_timestamp";
+    private static final String EXPIRY_TIMESTAMP_NAME = "expiry_timestamp";
+    private static final String MAX_TIMESTAMP_NAME = "max_timestamp";
+    private static final String TOKEN_ID_KEY_NAME = "token_id";
+    private static final String HMAC_KEY_NAME = "hmac";
+
+    private final Errors error;
+    private final long issueTimestamp;
+    private final long expiryTimestamp;
+    private final long maxTimestamp;
+    private final String tokenId;
+    private final ByteBuffer password;
+    private final int throttleTimeMs;
+    private KafkaPrincipal owner;
+
+    private static final Schema TOKEN_CREATE_RESPONSE_V0 = new Schema(
+        ERROR_CODE,
+        new Field("owner", STRING, "Token owner."),
```

Comment:
I was thinking it will be easy to construct DelegationToken object from response. I am not able to find easy way to get user principal on client side.  For now, added 2 separate fields for owner field.